### PR TITLE
Replace travis badge with github actions badge

### PR DESCRIPTION
**Pull request by maximiliankolb**

### CODE REVIEW

**Comment by nadjaheitmann** [comment]
LGTM.

**Comment by nadjaheitmann** [comment]
If I wanted to nitpick, I would say, that you could add one commit that replaces the travis badge with the github actions badge instead of having two separate commits.

**Review by nadjaheitmann — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
-[![Build Status master](https://travis-ci.org/ATIX-AG/foreman_scc_manager.svg?branch=master)](https://travis-ci.org/ATIX-AG/foreman_scc_manager)
+![Rubocop](https://github.com/ATIX-AG/foreman_scc_manager/actions/workflows/rubocop.yaml/badge.svg)
```
Comment (m-bucher) [inline README.md:1]: Either add the newly added pipelines from #74 or display an aggregated status, if github provides it.

**Review by m-bucher — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (maximiliankolb) [reply README.md:1]: Added the second badge.

**Review by m-bucher — APPROVED** [review]

**Pull request merged**